### ISSUE TITLE
Host application to allow live demo

### ISSUE THREAD

**Issue by dhuang612** [Arcaster42/rhymus#53]
I was thinking this would be a cool project for someone. 
Basically take the code as is and find somewhere to host it so that a link can be added to this project's page. Then people can click the link and play the game to so they have a visual of the project.

**Comment by Arcaster42** [comment]
I agree. Deployment is a critical part of development. The vanilla version will be simple enough to deploy that anyone could pick it up.
